### DISCUSSION TITLE
[Question] Why is FileBrowser using "X-Auth" header instead of more standard "Authorization: Bearer ..."

### DISCUSSION THREAD

**Discussion by erny** [filebrowser/filebrowser#1171] [Q&A]
Thanks for making this great work public.

**Is your feature request related to a problem? Please describe.**

I'm integrating FileBrowser's front-end with FastAPI and it seems that using X-Auth header is not very standard but using Authorization: Bearer ... header .

So my question is why it's not using the standard. I have changed it in the frontend for this integration and it seems easy to change.   

**Describe the solution you'd like**

Use "Authorization" header to make it more reusable.

Affected Files:
* http/auth.go
* src/api/files.js 
* src/api/utils.js
* src/utils/auth.js

**Describe alternatives you've considered**

None.

**Additional context**

I could provide PR.

Thanks a lot in advance.

**Comment by o1egl** [comment]
I think the idea was to use this header for different type of authentication mechanisms.
